gateway/ln-gateway/src/ng/mod.rs

```diff
@@ -531,3 +510,42 @@ impl State for GatewayClientStateMachines {
         }
     }
 }
+
+#[derive(Error, Debug, Serialize, Deserialize, Encodable, Decodable, Clone, Eq, PartialEq)]
+pub enum ReceiveError {
+    #[error("Route htlc error")]
+    RouteHtlcError,
+}
+#[derive(Debug, Clone, Eq, PartialEq, Decodable, Encodable)]
+pub struct Htlc {
```

Comment:
Did this code just move?

Reply:
Yes. from `ln-gateway/src/ng/incoming.rs` which no longer exists